### DISCUSSION TITLE
年齡與智慧之間並沒有關聯

### DISCUSSION THREAD

**Discussion by a3240281370** [1101-GS4719-C/Python-Community-GS4719-C#1411] [General]
不知道大家有沒有過被長輩倚老賣老的經驗，當下的內心真的一堆OS但又不能直接講出來
難道，吃的鹽越多講的話就越可信嗎？
之前就有一個科學研究打臉這個刻板印象，首先，科學家先給智慧一個公道的定義，他們認為有智慧的人應該要具備：
​1. 調節情緒
2. 反省自我
3. 解決社交衝突
4. 在經濟博弈中做出正確決策
5. 處理風險和不確定性的能力
接著，助理們會按照上述定義設計考題讓不同年齡的受測者寫看看。結果發現受測者的成績並不會隨年齡上升而增加，可見年齡和智慧間的關聯，趨近於零。
你以為下次被教授倚老賣老的時候我會用這個研究打臉他嗎？不會，我一樣會乖乖閉嘴....

**Comment by tower20021202** [comment]
在生活中我也常常會遇到長輩倚老賣老的時候，雖然會覺得很煩，但我依然會用友善包容的態度尊重他，如果不重要，自然就會左耳進右耳出，不讓他們對我造成影響

**Comment by Alexgns** [comment]
我認為智慧是一種綜合了知識、經驗和洞察力的能力，不僅取決於年齡，還取決於個人學習、成長和思考的方式。人們可以在任何年齡段發展和展現智慧，關鍵在於持續學習、開放思維和不斷挑戰自己。所以無論年齡多大，我們都應該給予每個人平等的機會和尊重，並認識到智慧是多元和豐富的。

**Comment by 1104030360** [comment]
我認為經驗還在某些情況下還是非常有用，畢竟他們有經歷過相同的事，我會選擇聽聽看建議，但要不要做的選擇權還是在我，畢竟現在的時代進步很快，舊時代的知識很快就會被淘汰。

**Comment by youtinglin** [comment]
我認為身處在亞洲社會不免除會遇到長輩對你倚老賣老，但是我個人在當下仍會去聽取他們的意見，畢竟還是需要給予長輩尊重，至於最後我會如何做，可能就會自行消化和分析再做決定。我之前也有遇到被教授倚老賣老的經歷，當下是很不知所措並且十分的不認同，但經過一段時間的冷靜思考，試著站在教授的角度想，確實覺得教授的說法也不會完全不理解，下次可能就會做更充足的準備，才能更有底氣的和教授討論，很多時候和各種人溝通都是一次次的學習，學會說話的藝術和理解不同思維角度。

**Comment by ALAN30359704** [comment]
我認為教授有時不一定是倚老賣老，因為他們的經驗確時比較豐富，而研究時常需要的是智慧、方法還有經驗，所以其實我覺得還是需要聽聽有經驗人的想法。

**Comment by alicia0725** [comment]
我認為年齡跟智慧是沒有太大的關聯的，長輩有些時候說的是他們曾經的生活經驗，但大部分或許已經不適用，但我覺得還是可以參考看看不一定都抱有偏見

**Comment by alstinlin** [comment]
我覺得智慧應該是跟經驗有比較大的關係，長輩時常被認為比較有智慧的原因，可能是他們在許多方面都累積不少經驗，但是在聽取建議的同時，因為長輩可能是從他的角度給出建議，所以自己也必須思考是不是合適的

**Comment by xiang305** [comment]
我認為年齡跟智慧頂多能算是正相關，真正有關聯的是經驗。俗話說：「經一事長一智」，隨著我們慢慢步入社會，眼界與經歷的事也不同以往，許多事我們都能依靠過去的經驗來解決。不過依靠過去經驗得出的結論並非完全正確的，有時還是得依靠自己的判斷來調整

**Comment by Aaron0810** [comment]
我是覺得有時候老人家或是家中長輩在看一些事情的時候還是比我們年輕人準，但畢竟年輕人會覺得長輩們在倚老賣老，長輩會覺得自己講的話比較有分量，導致雙方都不太信任對方，這也可能是原po講的那個測試會有那種結果的原因，所以我覺得只要相互傾聽，在某些時候長輩們說的話還是對的。

**Comment by Elaine2899** [comment]
或許因為時代變遷太快的關係，許多人的觀念會跟不上變遷，這樣會導致世代的隔閡。上述這也許是原因之一，但同時我也覺得很多人是馬齒徒長，只長了年紀和皺紋，經驗和歷練都沒有增加，這是我最不希望的未來，看到這些人我也會警惕自己不要和他們一樣。

**Comment by Evening218** [comment]
真正與年齡有關的是經驗,但我認為許多長輩的觀念雖然的確是來自他們多年來所經歷的事,但反而是這樣所以更難去接受變化,或者是不願意去了解新的時代跟觀念,我覺得如果長輩們願意試著去理解新事物,並跟自己的經驗結合,那麼年齡就會與智慧成正比了
